npc/controller_test.go

```diff
@@ -79,16 +79,22 @@ func (i *mockIPSet) entryExists(ipsetName ipset.Name, entry string) bool {
 }
 
 func (i *mockIPSet) Flush(ipsetName ipset.Name) error {
-	return errors.New("Not Implemented")
+	if _, ok := i.sets[string(ipsetName)]; !ok {
+		return errors.Errorf("ipset %s does not exist", ipsetName)
+	}
+	delete(i.sets, string(ipsetName))
+	return nil
 }
 
 func (i *mockIPSet) FlushAll() error {
 	return errors.New("Not Implemented")
 }
 
 func (i *mockIPSet) Destroy(ipsetName ipset.Name) error {
-	if _, ok := i.sets[string(ipsetName)]; !ok {
+	if set, ok := i.sets[string(ipsetName)]; !ok {
 		return errors.Errorf("ipset %s does not exist", ipsetName)
+	} else if len(set.subSets) != 0 {
```

Comment:
The len check is not required for hash:ip ipsets. Due to the check you see the unit test case failure at the CI.

Reply:
Yep just figured out. Testing PR in cluster :) thanks.